devel/cpphs/Portfile

```diff
@@ -4,23 +4,24 @@ PortSystem          1.0
 PortGroup           haskell_stack 1.0
 
 name                cpphs
-version             1.20.8
+version             1.20.9.1
+revision            0
 categories          devel haskell
 maintainers         nomaintainer
 license             GPL-3
 platforms           macosx
-homepage            https://www.haskell.org/alex/
+homepage            http://projects.haskell.org/cpphs/
```

Comment:
this is redirecting to `https://archives.haskell.org/projects.haskell.org/cpphs/`

Reply:
This is the URL specified as the Home Page at the upstream source: https://hackage.haskell.org/package/cpphs